ingest/src/main/java/com/microsoft/azure/kusto/ingest/result/ValidationPolicy.java

```diff
@@ -0,0 +1,68 @@
+package com.microsoft.azure.kusto.ingest.result;
+
+import java.io.Serializable;
+
+public class ValidationPolicy implements Serializable {
+    private ValidationOptions validationOptions = ValidationOptions.DoNotValidate;
+    private ValidationImplications validationPolicyType = ValidationImplications.BestEffort;
+
+    public ValidationPolicy() {
+    }
+
+    public ValidationPolicy(ValidationOptions validationOptions, ValidationImplications validationPolicyType) {
```

Comment:
What if they want to provide just 1 or the other?

Reply:
I guess they should then just use the default constructor followed by the appropriate setter, NBD.

Reply:
So I'm resolving.